lecture/urls.py

```
@@ -1,9 +1,10 @@
 from django.conf.urls.static import static
 from django.conf import settings
-urlpatterns=[
-    path('home',views.lecture_home ,name="lecture_home"),
-    path('create_topic',views.create_title,name="create_topic"),
-    path('create_lecture',views.create_lecture,name="create_lecture")
-]
\ No newline at end of file
+
+urlpatterns = [
+    path('home/<str:course_id>', views.lecture_home, name="lecture_home"),
+    path('video/<str:video_id>',views.lecture_video,name="lecture_video_content")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)